packages/web_drivers/lib/chrome_driver_installer.dart

```diff
@@ -78,7 +75,7 @@ class ChromeDriverInstaller {
     if (chromeDriverVersion.isEmpty) {
       final int chromeVersion = await _querySystemChromeVersion();
```

Comment:
It seems that at some point, _querySystemChromeVersion might have returned null? Is this `int?` ?

Reply:
It seems to know if it's now able to find a version.